doc/modules/preprocessing_targets.rst

```diff
@@ -7,6 +6,71 @@
 Transforming the prediction target (``y``)
 ==========================================
 
+Transforming target in regression
+---------------------------------
+
+:class:`TransformedTargetRegressor` is a meta-estimator transforming the target
+before fitting a regression model and inverting back the prediction to the
```

Comment:
I'm not sure I like the formulation, but I'm not sure I have a better one... "and transforming the model predictions back to the original space before outputting them"?